### ISSUE TITLE
sonar-scanner-cli:5.0.1 shows Alpine 3.17 (not 3.19)

### ISSUE THREAD

**Issue by Trikenstein** [SonarSource/sonar-scanner-cli-docker#231]
[Dockerfile](https://github.com/SonarSource/sonar-scanner-cli-docker/blob/master/5/Dockerfile) shows `FROM alpine:3.19`

The image from [DockerHub](https://hub.docker.com/r/sonarsource/sonar-scanner-cli/tags), tag = `latest` (or `5.0.1`) as of 2024-04-28  shows Alpine is actually 3.17.7

```
docker run -it --entrypoint sh sonarsource/sonar-scanner-cli:5.0.1
```

From within the docker image

```
/usr/src # cat /etc/os-release

NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.17.7
PRETTY_NAME="Alpine Linux v3.17"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://gitlab.alpinelinux.org/alpine/aports/-/issues"
```

**Comment by 7PH** [comment]
Hey @Trikenstein , 

From what I can see, [sonar-scanner-cli/5.0.1](https://hub.docker.com/layers/sonarsource/sonar-scanner-cli/5.0.1/images/sha256-02372948eaeeb10dfbe0cfd4174d44b8e405d0aeae431532b2bdb21d0347bf23?context=explore) was released 13 days ago (April 16th) along with [the associated GitHub release](https://github.com/SonarSource/sonar-scanner-cli-docker/releases/tag/5.0.1.3006). 

If you look at the [version of the Dockerfile which was shipped](https://github.com/SonarSource/sonar-scanner-cli-docker/blob/df59f80f4521b0ea86b0916be92c987f460d93cc/5/Dockerfile), the base image is indeed `alpine:3.17`. 

When we release the next version of this project, the version of alpine specified on master branch (currently `alpine:3.19`) will be used

Hope that helps, I'll go ahead and close this issue as resolved

**Comment by Trikenstein** [comment]
It's rather surprising a code in `master` is not the same as the release. In my opinion, if you hold off a release, the pending change should rather stay in a feature branch. When all tests passed and deemed OK for release, at that time, merge that feature branch  to master and create a release at the same time.

**Comment by groja** [comment]
@7PH there seems to be a later release in github (https://github.com/SonarSource/sonar-scanner-cli-docker/releases/tag/5.0.1.3006-update-base-image) which does have this update, is there a timeline on when that will push to Docker Hub? Alpine 3.17 contains a vulnerability that some scanners flag as critical, which is resolved in 3.19, so an image based on 3.19 is very desireable!

**Comment by 7PH** [comment]
Hey guys, thanks for getting back, you are correct @groja , indeed `alpine:3.19` isn't in the `5.0.1.3006` release because there's a later release (https://github.com/SonarSource/sonar-scanner-cli-docker/releases/tag/5.0.1.3006) which reverted changes from the previous one. This reverted the base image to use alpine `3.17` instead of `3.19`.

We are prioritizing making a new release that you can expect this month. but we do not have a fixed date for it yet. You could watch/star this repo to get notified when that happens.
